boards/Parrot/Parrot.config

```diff
@@ -0,0 +1,55 @@
+# Configuration for a x230 running Qubes and other OSes
+# STATIC_OPTION_TABLE is set inside of coreboot config
```

Comment:
Not sure required. IF not, please modify

Reply:
sorry, modify to what ?

Reply:
@gompa if related optionsnot required in coreboot, remove there and here.